18-hlt-intro.md

```diff
@@ -32,15 +32,24 @@ lines are saved to the raw event in a stripped-down form. One of the things that
 is saved are all the LHCbIDs of the final state particles of a decay tree.
 LHCbIDs are unique identifiers for detector hits.
 
-We will now have a look at the LHCbIDs of some of the stored candidates. We will
-use the script we [created earlier](http://lhcb.github.io/first-analysis-steps/05-interactive-dst.html) as a starting
-point, and the file you [downloaded before](http://lhcb.github.io/first-analysis-steps/05-files-from-grid.html). Fire up
-your favourite editor, open the script and save a copy to work on as
-`tistos.py`. There are a few things in the script that we don't need and can be
-removed, such as the print_decay and decay finder tools.
+A new feature in Run II is the so-called Turbo stream. Since the reconstruction
+available in HLT2 is the same as the offline reconstruction, physics analysis
+can be done with the candidates created in HLT2. If a line is configured to be a
+Turbo line, all information on the candidates that it selects is stored in the
+raw event. These candidates can be resurrected later by the Tesla application and
+written to a microDST.
+
+We will now have a look at some of the candidates stored by the HLT. We will use the script we
+[used last time](http://lhcb.github.io/first-analysis-steps/05-interactive-dst.html)
+as a starting point, and the file
+`root://eoslhcb.cern.ch//eos/lhcb/user/r/raaij/Impactkit/00051318_00000509_1.turbo.mdst`.
+This file contains some 2016 Turbo events from run 174252.  Fire up your
```

Comment:
Might be nice to link to the [RunDB for 174252](http://lbrundb.cern.ch/rundb/run/174252/).